src/status_im/contexts/wallet/send/routes/view.cljs

```diff
@@ -2,31 +2,52 @@
-    [utils.i18n :as i18n]))
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn route-item
+  [{:keys [amount from-network to-network status]}]
+  [rn/view {:style style/routes-inner-container}
+   [quo/network-bridge
+    {:amount  amount
+     :network from-network
+     :status  status}]
+   [quo/network-link
+    {:shape           :linear
+     :source          from-network
+     :destination     to-network
+     :container-style style/network-link}]
+   [quo/network-bridge
+    {:amount          amount
+     :network         to-network
+     :status          status
+     :container-style {:right 12}}]])
 
 (defn view
-  [{:keys [amount from-network to-network]}]
-  [rn/view {:style style/routes-container}
-   [rn/view {:style style/routes-header-container}
-    [quo/section-label
-     {:section         (i18n/label :t/from-label)
-      :container-style (style/section-label 0)}]
-    [quo/section-label
-     {:section         (i18n/label :t/to-label)
-      :container-style (style/section-label 64)}]]
-   [rn/view {:style style/routes-inner-container}
-    [quo/network-bridge
-     {:amount  amount
-      :network from-network
-      :status  :default}]
-    [quo/network-link
-     {:shape           :linear
-      :source          from-network
-      :destination     to-network
-      :container-style style/network-link}]
-    [quo/network-bridge
-     {:amount          amount
-      :network         to-network
-      :status          :default
-      :container-style {:right 12}}]]])
+  [{:keys [amount routes]}]
+  (let [loading-suggested-routes? (rf/sub [:wallet/wallet-send-loading-suggested-routes?])
+        candidates                (:Candidates routes)]
+    (if (and (not loading-suggested-routes?) (not-empty candidates))
+      [rn/flat-list
+       {:data                    candidates
+        :content-container-style style/routes-container
+        :header                  [rn/view {:style style/routes-header-container}
+                                  [quo/section-label
+                                   {:section         (i18n/label :t/from-label)
+                                    :container-style (style/section-label 0)}]
+                                  [quo/section-label
+                                   {:section         (i18n/label :t/to-label)
+                                    :container-style (style/section-label 64)}]]
+        :render-fn               (fn [route]
+                                   [route-item
+                                    {:amount       amount
+                                     :status       :default
+                                     :from-network (utils/id->network (get-in route [:From :chainId]))
+                                     :to-network   (utils/id->network (get-in route [:To :chainId]))}])}]
+      [rn/view {:style style/empty-container}
+       (if loading-suggested-routes?
+         [rn/activity-indicator]
```

Comment:
Using `activity-indicator` until the design team confirms what should the UI look like while fetching the suggested routes.

Reply:
Not sure if is this the one you are referring too. 

<img width="200" src="https://github.com/status-im/status-mobile/assets/19339952/f306fd66-4f9b-4e64-858d-4e34f99b3347" />

[Figma Link](https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=7701%3A157490&mode=design&t=G8Y5fc4xLP9AWVcK-1)

Reply:
@smohamedjavid yes this is the loading routes which is implemented in that PR https://github.com/status-im/status-mobile/pull/18279#issue-2053556526

You can check the linked comment. The problem here is that the UI is assuming we know the routes while loading.